src/services/WebViews.ts

```diff
@@ -88,6 +89,18 @@ export class WebViews {
 
 			const resultStates: string[] = await this.httpClient.getRequest(`sast/result-states`);
 			query.resultStates = resultStates;
+
+			let usersResponse = await this.httpClient.getRequest(`auth/AssignableUsers`);
```

Comment:
Did you  test this API by logging-in with different user that belong to some other team than /CxServer? The assignable users list should vary based on the team (s) to which logged-in user belong.

Reply:
This list can be fetched only once, not everytime results table is refershed when  High/Medium/low or child nodes on the tree are clicked.